ProcessMaker/Policies/ProcessPolicy.php

```diff
@@ -35,8 +36,10 @@ public function before(User $user)
      */
     public function start(User $user, Process $process)
     {
-        $groupIds = $user->groups->pluck('id');
-
+        $userGroupIds = $user->groups->pluck('id')->all();
+        $nestedGroupIds = GroupMember::whereIn('member_id', $userGroupIds)->pluck('group_id')->all();
```

Comment:
I think we need to add another `where` here to ensure that the `member_type` is `ProcessMaker\Models\Group` otherwise we may end up getting user IDs in this query as well.